packages/ember-data/tests/unit/model_test.js

```diff
@@ -649,6 +649,20 @@ test("A subclass of DS.Model can not use the `data` property", function() {
   }, /`data` is a reserved property name on DS.Model objects/);
 });
 
+test("A subclass of DS.Model can not use the `store` property", function() {
+  var Retailer = DS.Model.extend({
+    store: DS.attr(),
+    name: DS.attr()
+  });
+
+  var store = createStore({retailer: Retailer});
```

Comment:
Looks like travis is complaining because jscs wants spaces after the opening brace and before the closing brace. 

`({ retailer: Retailer });`